apps/nextjs-website/src/components/atoms/TabComponent/TabComponent.tsx

```diff
@@ -0,0 +1,57 @@
+'use client';
+import React, { FC, ReactNode } from 'react';
+import { Box, Tab, Tabs, Typography } from '@mui/material';
+
+export type TabPanelProps = {
```

Comment:
Why do you export this type?
```suggestion
type TabPanelProps = {
```